diesel_migrations/migrations_internals/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "migrations_internals"
```

Comment:
I don't understand the need for this crate. Everything in here was previously part of the public API, and I don't see the reasoning for making it private. Why does this need to be separate from `diesel_migrations`?

Reply:
We need a extra crate here because some parts of this are used [inside of migrations_macros](https://github.com/diesel-rs/diesel/pull/1231/files#diff-d004f1c44c15da1ffeae40bf38f49fafR12). Everything that was public before inside the migrations module remains public, because it is [reexported](https://github.com/diesel-rs/diesel/pull/1231/files#diff-27119e6fe3360c1e2e3dd6a25aa5e811R19) through diesel_migrations